elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideAsyncConfiguration.java

```diff
@@ -105,4 +102,23 @@ public AsyncQueryDAO buildAsyncQueryDAO(Elide elide) {
         Elide asyncElide = new Elide(asyncElideSettings);
         return new DefaultAsyncQueryDAO(asyncElide, asyncElide.getDataStore());
     }
+
+    /**
+     * Configure the AsyncQueryDAO used by async query requests.
+     * @param elide elideObject.
+     * @return an AsyncQueryDAO object.
+     */
+    @Bean
+    @ConditionalOnMissingBean
+    @ConditionalOnProperty(prefix = "elide.async", name = "defaultResultStorageEngine", matchIfMissing = true)
+    public ResultStorageEngine buildResultStorageEngine(Elide elide) {
+        // Creating a new ElideSettings and Elide object for Async services
+        // which will have ISO8601 Dates. Used for DefaultAsyncQueryDAO.
+        ElideSettings asyncElideSettings = new ElideSettingsBuilder(elide.getDataStore())
```

Comment:
L117 to L121 is common between the methods in this class. Instead of duplicating we can move to a private method and use it.